tutorials/javaFxPart3.md

```diff
@@ -26,38 +26,40 @@ When an event is detected, JavaFX will call the respective handlers that was pre
 
 For Duke, there are two events that we want to respond to, namely the user pressing `Enter` in the `TextField` and left-clicking the `Button`. These are the `onAction` event for the `TextField` and the `onMouseClicked` event for the `Button`.
 
-For now, let’s have the application add a new `Label` with the text from the `TextField`. Update the `Main` class as follows. You'll need to add an `import javafx.scene.control.Label;` too.
+For now, let’s have the application add a new `DialogBox` with the text from the `TextField`, so the user can see the message they have sent. Update the `Duke` class as follows.
 ```java
 @Override
 public void start(Stage stage) {
-    // Step 1 code here
+    //Step 1 code here
 
     //Step 2 code here
+    //Delete or comment off the following section
```

Comment:
`Delete or comment off` is a bit dodgy. Commenting is fine only if the change is temporary. Otherwise, it should be deleted.